actor/sharedaction/is_logged_in_test.go

```diff
@@ -0,0 +1,65 @@
+package sharedaction_test
+
+import (
+	. "code.cloudfoundry.org/cli/actor/sharedaction"
+	"code.cloudfoundry.org/cli/actor/sharedaction/sharedactionfakes"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("IsLoggedIn", func() {
+	var (
+		actor      *Actor
+		binaryName string
+		fakeConfig *sharedactionfakes.FakeConfig
+	)
+
+	BeforeEach(func() {
+		binaryName = "faceman"
+		fakeConfig = new(sharedactionfakes.FakeConfig)
+		fakeConfig.BinaryNameReturns(binaryName)
```

Comment:
It doesn't look like binary name is needed. If I comment out this line (and others relating to binaryName), the test still passes for me.

Same goes for IsOrgTargeted and IsSpaceTargeted below.